Dockerfile

```diff
@@ -56,6 +56,15 @@ COPY scripts/dummy.rs libafl_targets/src/lib.rs
 COPY libafl_tests/Cargo.toml libafl_tests/build.rs libafl_tests/
 COPY scripts/dummy.rs libafl_tests/src/lib.rs
 
+COPY libafl_concolic/test/dump_constraints/Cargo.toml libafl_concolic/test/dump_constraints/
+COPY scripts/dummy.rs libafl_concolic/test/dump_constraints/src/lib.rs
```

Comment:
Ah damnit. It should be a `main.rs`, right?

Reply:
The idea in the dockerfile is currently:
copy a dummy lib, install dependencies, then copy the actual lib over later
that way, changing the lib.rs won't trigger a full docker rebuild

Reply:
Okay, I think I'm starting to understand what this is supposed to do. Before, I had just imitated what came before it.

I can't really remove the COPY commands here, because it would stop building. What should I do instead? I'm a bit lost...

Reply:
Copy over everything you need?